### DISCUSSION TITLE
Publish / unpublish UX with drafts, should be reworked.

### DISCUSSION THREAD

**Discussion by quornik** [payloadcms/payload#9103] [Feature Requests & Ideas]
## **I wanna talk about how the publish / unpublish and drafts system works.**

You can't simply unpublish document, after changing it and saving a draft.
to be precise: Technically you can do it, but you need to :
- Go to versions and revert document to currently published version,
- Then unpublish it,
- Then go back to versions and revert to the version containing drafts you made (before reverting to published version, just to unpublish it...)

This is propably logical for people using GIT. And propably only for people using GIT. :)

If you are enduser, who worked with Wordpress before it is not.

It is confusing and hard to defend to endclient, that it should work this way.
---
Currently in UI, if I make changes to document, the unpublish button in replaced with revert to published. 

**It should't be replaced.** 

The **revert to published should appear as a second functionality**, and the ability to unpublish **should always be available** as long as there is any published version of the document out there.

**Comment by timothylp** [comment]
I fully agree with this. The current unpublish flow becomes unintuitive for non-technical users, especially clients coming from platforms like WordPress. Overwriting the “Unpublish” button with “Revert to published” introduces unnecessary friction and confusion. Ideally, the “Unpublish” action should remain accessible regardless of the draft state. This would better align with common CMS usage patterns and improve the overall UX for content editors.
